wrappers/csharp/Utils.cs

```diff
@@ -20,8 +27,15 @@ public static string ToBase64String(byte[] bytes)
             {
                 return null;
             }
-
-            return Convert.ToBase64String(bytes);
+            
+            try
```

Comment:
The only listed exception is `ArgumentNullException` which we check for, I don't think we need this try-catch.

Reply:
Fixed